packit/upstream.py

```diff
@@ -1489,7 +1494,10 @@ def _fix_specfile_to_use_local_archive(
             archive: Path to the archive.
             update_release: Should Release be updated?
             release_suffix: Append this suffix to the %release.
+            preserve_spec: Should spec file be preserved without updating?
         """
+        if preserve_spec:
+            return
```

Comment:
Wouldn't it make more sense not to call this method at all if `preserve_spec` is `True` (i.e. moving the check to the caller)?

Reply:
Thanks, it indeed does. I'll change it.